paasta_tools/monitoring_tools.py

```diff
@@ -400,14 +401,21 @@ def check_smartstack_replication_for_instance(
 
 def send_replication_event_if_under_replication(
     instance_config,
-    expected_count,
-    num_available,
+    expected_count: int,
+    num_available: int,
+    subinstance_type: Optional[str] = None,
```

Comment:
maybe `sub_component` could be a better word?

Reply:
done